docs/test/layouts/detail-highlight.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Detail Highlight Test Page
+layout: DetailHighlight
+frameOverrides:
+  header:
+    homeLink: /mosaic/index
+    title: Mosaic (BETA)
+    logo: /img/favicon.png
+    searchNamespace: mosaic
+    menu:
+      - title: Docs
+        link: /mosaic/docs/index
+      - title: Quick Start
+        link: /mosaic/quick-start/index
+  footer:
+    description: Footer Content
+    title: Detail Highlight Layout Footer
+    href: /mosaic
+breadcrumbs:
+  - label: Breadcrumb 1
+    path: /mosaic/index.mdx
+    id: /mosaic/index.mdx
+  - label: Breadcrumb 2
+    path: /mosaic/index.mdx
+    id: /mosaic/index.mdx
+  - label: Breadcrumb 3
+    path: /mosaic/index.mdx
+    id: /mosaic/index.mdx
+navigation:
+  prev:
+    title: Previous Page
+    route: /mosaic/docs/modes/index
+  next:
+    title: Next Page
+    route: /mosaic/docs/modes/snapshot-file
+---
+
+## {meta.title}
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
+
+## TOC 1
```

Comment:
Would be better to snapshot the names as `Heading 2`, `Heading 3` rather than `TOC`
Then it keeps the naming clear, TOC is a component that shows the headings on a page